runtime/convertValues.go

```diff
@@ -550,6 +561,45 @@ func importPathValue(v cadence.Path) interpreter.PathValue {
 	}
 }
 
+func importTypeValue(
+	_ *interpreter.Interpreter,
+	v cadence.Type,
+) (
+	interpreter.TypeValue,
+	error,
+) {
+	return interpreter.TypeValue{
+		Type: ImportType(v),
+	}, nil
+}
+
+func importCapability(
+	_ *interpreter.Interpreter,
+	path cadence.Path,
+	address cadence.Address,
+	borrowType cadence.Type,
+) (
+	*interpreter.CapabilityValue,
+	error,
+) {
+
+	_, ok := borrowType.(cadence.ReferenceType)
+
+	if !ok {
+		return &interpreter.CapabilityValue{}, fmt.Errorf(
```

Comment:
It's more idiomatic to return nil, which also avoids an unnecessary allocation: 

```suggestion
		return nil, fmt.Errorf(
```